MSOpenTech/vagrant-azure
# Update to the latest version of the ruby sdk for Azure

- update to the latest version of the ruby sdk for azure
-- includes updates to all the machine sizes available in azure
-- provides more consistent http client support
-- provides more options for specifying management certificates (pfx, pem, and raw strings)
- update docs with better documentation of ssh configuration and private key support
- update winrm support with the 1.3.3 winrm gem
- remove confusing certificate_file_path, which wasn't used

should fix #88, #86, #83, #77, #76, #75
